src/module.c

```diff
@@ -11486,6 +11486,14 @@ int RM_GetDbIdFromDefragCtx(RedisModuleDefragCtx *ctx) {
     return ctx->dbid;
 }
 
+char *RM_DumpACL() {
+    return saveACLToBuffer();
+}
+
+char *RM_LoadACL(char *aclString) {
```

Comment:
I don't think the return of loadACLFromBuffer is a char *, it looks like an sds. loadACLFromBuffer also expects an sds and you are passing in a char *

Reply:
yes "sds" (which seems to be just a char * behind the scene).  I wasn't sure how to expose the "sds" to redis modules, so just stuck with the char *

Reply:
adjusted to take a RedisModuleString and extract the sds from it.

Reply:
Just for context, sds is not a char * behind the scenes. SDS also contains metadata about the length of the string, but can be used as if it was a C string (I.E. null terminated).